org.webonnix.beamprivace.yaml

```diff
@@ -0,0 +1,23 @@
+app-id: org.beamprivacy.desktop
+runtime: org.kde.Platform
+runtime-version: '3.1.5765'
```

Comment:
I'm sure it's not what you want. It is for specifying runtime version, not application version.